web/src/test/java/com/epam/rft/atsy/web/mapper/ExceptionMessagePairValidationExceptionMapperImplTest.java

```diff
@@ -10,7 +10,7 @@
 import java.util.HashSet;
 import java.util.Set;
 
-public class RuleValidationExceptionMapperImplTest {
+public class ExceptionMessagePairValidationExceptionMapperImplTest {
```

Comment:
Why did you rename this class? In this test class you test RuleValidationExceptionMapperImpl class, as a 
convention the test class name is RuleValidationExceptionMapperImplTest.